runtime/doc/options.txt

```diff
@@ -187,7 +187,7 @@ opt+=val" the expansion is done before the adding or removing.
 Handling of local options			*local-options*
 
 Some of the options only apply to a window or buffer.  Each window or buffer
-has its own copy of this option, thus can each have their own value.  This
+has its own copy of this option, thus each have its own value.  This
```

Comment:
in the vim patch it is "thus each can have its own value"

Reply:
I even read over that a couple times, because the wording seemed odd...